bootstrap.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+apt-get update
+apt-get install python-software-properties -y
+apt-add-repository ppa:brightbox/ruby-ng -y
+apt-get update
+apt-get install ruby2.3 ruby2.3-dev mongodb-server make -y
+gem install bundler
```

Comment:
I intentionally didn't use any more advanced provisioning because there is no need for it yet, as you can see this file has just few lines.
There are still many decisions that need to be made before we can continue with this project. This is just to make it easier to start with what we have so far.